docs/reference_keymap_extras.md

```diff
@@ -63,6 +63,7 @@ There are also a few which are not quite language-specific, but useful if you ar
 |-------------------|------------------------|
 |Colemak            |`keymap_colemak.h`      |
 |Dvorak             |`keymap_dvorak.h`       |
+|Dvorak (French)    |`keymap_dvorak_fr.h`    |
```

Comment:
```suggestion
|Dvorak (French)*   |`keymap_dvorak_fr.h`    |
```

Reply:
I add the sendstring file, the star is not necessary